plugins/action/jcliff.py

```diff
@@ -70,8 +70,12 @@ def _lookup_subsys_template(self, subsys_name):
 
     def _build_and_deploy_jcliff_rule_files(self, tmp_remote_src):
         display.vvvv(u"Build and deploy jcliff rule tmpfile: %s" % tmp_remote_src)
-        display.vvvv(u"Subsystems: %s" % self._task.args['subsystems'])
-        subsystems = self._task.args['subsystems']
+
+        if 'subsystems' in self._task.args:
```

Comment:
By specifying the alias, both options should map to the `components` variable so we should not need the conditional check

Reply:
I've tried, did not work... Maybe you can try and let me know if it works for you?

Reply:
it works for me without issue

Reply:
CI seems to disagree with your statement. The error is that there is no rule file generated for jcliff (which means the if 'subsystems' in self._task.args: was false and the all rule generation was skipped).

Can you double check ? And maybe update the PR ?

Reply:
@rpelisse heres a superset of this PR -> https://github.com/wildfly-extras/ansible_collections_jcliff/pull/46